plugins/buffer/README.md

```diff
@@ -41,7 +41,7 @@ gets transferred to the destination).
 
 <div>
```

Comment:
I found how to fix it.  Removing this `<div>` tag and L46 `</div>` tag will fix.

Reply:
ah, thanks for pointing out.
I updated the PR.